docs/spatial-vector.rst

```diff
@@ -4,6 +4,45 @@ Earthpy Spatial Vector Data
 The ``earthpy`` spatial module provides functions that wrap around the ``rasterio``
 and ``geopandas`` to work with raster and vector data in Python.
 
+Earthpy Spatial Vector Data
+===========================
+
+The ``earthpy`` spatial vector module provides functions that wrap around ``geopandas`` to work with vector data in Python.
+
```

Comment:
Lines 7-11 look like repeated lines from the lines above (1-6), so please remove them.